go-runtime/cmd/ftl-language-go/main.go

```diff
@@ -4,20 +4,93 @@ import (
 	"context"
 	"os"
 
+	"connectrpc.com/connect"
+	"github.com/alecthomas/kong"
+
+	"github.com/block/ftl"
+	languagepb "github.com/block/ftl/backend/protos/xyz/block/ftl/language/v1"
 	"github.com/block/ftl/backend/protos/xyz/block/ftl/language/v1/languagepbconnect"
 	"github.com/block/ftl/common/plugin"
 	"github.com/block/ftl/go-runtime/goplugin"
+	"github.com/block/ftl/internal/clirpc"
 )
 
+var cli struct {
+	Version kong.VersionFlag `help:"Show version."`
+	Serve   serve            `cmd:"" default:"1" help:"Run the service."`
+
+	GetNewModuleFlags       GetNewModuleFlagsCmd       `cmd:"" help:"Get language specific flags for a new module."`
+	NewModule               NewModuleCmd               `cmd:"" help:"Create a new module."`
+	GetModuleConfigDefaults GetModuleConfigDefaultsCmd `cmd:"" help:"Get the default config for a module."`
+}
+
+type serve struct {
+}
+
 func main() {
+	kctx := kong.Parse(&cli,
+		kong.Description(`FTL - Go`),
+		kong.UsageOnError(),
+		kong.Vars{"version": ftl.FormattedVersion},
+	)
+	kctx.FatalIfErrorf(kctx.Run())
+}
+
+func (s serve) Run() error {
 	plugin.Start(context.Background(),
 		os.Getenv("FTL_NAME"),
```

Comment:
I realise this is not new, but can we not use a name that is basically meaningless for this envar? "FTL_NAME" could mean literally anything!

Also, use Kong to provide envars so they're discoverable:

```go
  Name string `env:"FTL_NAME" help:"Name of the plugin provided by the plugin host."`
```